MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +287,30 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        let threshold = 1000.0 //meters
+        let zoomOutAltitude = 2000.0
+        let zoomInAltitude = 1000.0
```

Comment:
Is this the default altitude? Do we have a constant elsewhere for this?

Reply:
I observed the default altitude (aka I didn't do any gesture manipulation to the camera) to be ~1000 when doing testing this morning. If there is a default defined somewhere, it would probably be trivial to reference that.